### ISSUE TITLE
Overlay: Question before allowing it's use...

### ISSUE THREAD

**Issue by Bgolden67** [wet-boew/wet-boew#6763]
Has this been vetted for accessibility and usability? I ask only because in WET3 there was a vaguely similar plugin that was used for a table of contents but it wasn't very well received by the Accessibility group, and I believe there were some usability concerns as well. I would just like to ensure these are an issue with this overlay plugin and it will be safe to use going forward.

**Comment by pjackson28** [comment]
Yes, the overlay plugin has undergone accessibility and usability testing (including user testing). The floating TOC plugin from WET3 is an entirely different plugin and was decommisioned in WET4  because of the issues with it.

**Comment by Bgolden67** [comment]
Good to know! thanks!

**Comment by Bgolden67** [comment]
Sorry i just discovered that this plugin doesn't have a fallback in the event that javascript is disabled.. i know it's a lot more rare these days than in the past but with no javascript, what is in the overlay is now hidden from all users...

**Comment by LaurentGoderre** [comment]
@Bgolden67 you could ad a `<noscript>` section to add a fallback

**Comment by pjackson28** [comment]
Note that in WCAG 2.0, there is no longer a requirement to provide fallbacks for when JavaScript is disabled. That being said, PR #6766 will make the overlay visible at the end of the content area when JS is disabled.

**Comment by Bgolden67** [comment]
Thanks again guys... I thought of the no script as well, which is good for those who know, but having the fall back already there can't be a bad idea.. but i guess it's one of those old habits that had been drilled into us during CLF days ;-)
